components/chart/src/selection-area/selection-area.ts

```diff
@@ -456,6 +457,11 @@ export class DtChartSelectionArea implements AfterContentInit, OnDestroy {
 
   /** If there is an overlay open it will dispose it and destroy it */
   private _closeOverlay(): void {
+    // remove class showing the arrows of the range handles
+    if (this._chart._range) {
+      this._chart._range._reflectRangeReleased(false);
```

Comment:
why is this needed?

Reply:
@peter-affenzeller I tried to finish your PR as requested, but when I removed this part of the code, the test failed. Did you figure out, why this is needed?

Reply:
@lukasholzer @tomheller this is intended to remove the class that makes the little arrows of the drag handles fade in so that they fade in every time the range is opened (also with keyboard input. currently, when you focus the chart and convert a timestamp to a range they don't show), in combination with the reflectRangeReleased(true) in line 334. but I just checked again, this also seems to not work properly, probably better to do this in another PR.